src/main/java/com/google/sps/servlets/DataFromDatabaseServlet.java

```diff
@@ -42,7 +42,11 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    //TODO: check to make sure there is userid or deviceid in request
+    String userID = request.getParameter("user_id");
```

Comment:
Weren't we changing these to UserId and DeviceId to avoid the custom logic on the server?

Reply:
Yes, done now